app/views/user_mailer/notify_post_owner.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Hello, <%= @user.full_name %></h1>
+    <p>
+      There is a new comment by <%= @comment.user.full_name %> on your post <b><%= @comment.post.title %></b>
+    </p>
+    <p> To see this comment, <%= link_to "Click here", post_url(@comment.post) %> </p>
+    <p>Have a great day!</p>
+  </body>
```

Comment:
Maybe we should add a link to the profile settings in the email footer, explaining that one can disable notifications from there if they want to.